src/smc-webapp/projects.cjsx

```diff
@@ -774,6 +765,67 @@ NewProjectCreator = rclass
         else if @state.state == 'saving'
             <Alert bsStyle='info'>Working hard to build your project... <Icon name='circle-o-notch' spin /></Alert>
 
+    create_project_with_members_and_internet: ->
+        total_upgrades = redux.getStore('account').get_total_upgrades() ? {}
+        if total_upgrades.member_host > 0 and total_upgrades.network > 0
+            @create_project({member_host: 1, network: 1})
+        else
+            @setState(create_button_hit: 'with_members_and_internet')
+
+    render_upgrade_buttons: ->
+        <ButtonToolbar>
+            <Button
+                disabled = {@state.title_text == '' or @state.state == 'saving'}
+                bsStyle  = 'success'
+                onClick  = {=>@create_project(false)} >
+                Create project without upgrades
+            </Button>
+            <Button
+                disabled = {@state.title_text == '' or @state.state == 'saving' or @state.create_button_hit == 'with_members_and_internet'}
+                bsStyle  = 'success'
+                onClick  = {=>@create_project_with_members_and_internet()} >
```

Comment:
This unexpectedly creates a project even if I don't have members hosting spots available. This is because the function only checks if I have some upgrades in members and internet but not if they're available.